### ISSUE TITLE
CiscoSMB false positives from “show bootvar”

### ISSUE THREAD

**Issue by quacktacular** [ytti/oxidized#1500]
Recently I’ve been seeing an issue where the actual command “show bootvar” will be present in the diff instead of the actual output. This is almost always resolved on the next scan but produces a lot of notifications.

Happening on multiple SG500 switches. Short of removing the command from the model how can I troubleshoot further?

**Comment by davama** [comment]
could you provide a diff?
More info would be helpful

This is how my `sh bootvar` looks: (cisco sf300/302)
```
sh bootvar 
Image  Filename   Version     Date                    Status
-----  ---------  ---------   ---------------------   -----------
1      image-1    1.3.7.18    12-Jan-2014  18:02:59   Active* 
2      image-2    1.3.7.18    12-Jan-2014  18:02:59   Not active

"*" designates that the image was selected for the next boot
```

**Comment by quacktacular** [comment]
Sure no problem:
```
Version 155 (3 days 4 hours ago)
diff --git a/sw1.me.acme.com b/sw1.me.acme.com
index bc9b76f..bb923a0 100644
--- a/sw1.me.acme.com
@@ -4,8 +4,7 @@
 !          1                1.4.8.6            1.4.0.02               V05         
 !          2                1.4.8.6            1.4.0.02               V05         
 ! 
+! show bootvar
+! 
Unit  Image  Filename   Version    Date                    Status
 ! ----  -----  ---------  ---------  ---------------------   -----------
 ! 1     1      image-1    1.4.5.02   20-Apr-2016  12:24:28   Not active
 ! 1     2      image-2    1.4.8.6    10-Jul-2017  17:07:52   Active* 
```
Also do you know wether there's a way to suppress port status updates like this? Also causing notifications sometimes. 
```
Version 145 (1 hours 50 min ago)
diff --git a/sw5.me.acme.com b/sw5.me.acme.com
index 0e439c4..854e667 100644
--- a/sw5.me.acme.com
@@ -1,4 +1,3 @@
+! 30-Aug-2018 05:55:52 %LINK-W-Down:  gi1/1/44
 ! 
 !        Unit             SW version         Boot version         HW version      
 ! ------------------- ------------------- ------------------- ------------------- 
```

**Comment by davama** [comment]
That's interesting.

So this gets added/removed on every `sh bootvar` run?
```
+! show bootvar
+! 
```

> Also do you know wether there's a way to suppress port status updates like this? Also causing notifications sometimes.

you could use the gsub block to modify the output. something like below
https://github.com/ytti/oxidized/blob/40cd2ec8986d6c2040c725df8f6f84901cef6f2c/lib/oxidized/model/ciscosmb.rb#L16

**Comment by davama** [comment]
btw, what/why are you running to get the port status? isn't that what monitoring systems are for?

**Comment by quacktacular** [comment]
Thanks! Its intermittent. Happens once or twice a day, with oxidized on an hourly schedule.

**Comment by quacktacular** [comment]
Not running anything to get port status actually! Those just appears in the console for our SG switches any time you're connected via console / SSH. Think that's on by default but I could see if there is a way to disable.

**Comment by quacktacular** [comment]
Confirmed this has been happening because of various status / debug messages being printed to console during the oxidized run. Its weird because my switches don't have any configuration for console logging (its all default).

I copied the latest ciscosmb.rb model and added post_login commands to enter config mode, then "logging console emergencies" (instead of the default which is informational or debug presumably) and so far so good. 

Not sure why this wasn't a problem before? Either something has changed in oxidized or I have elevated errors in my network haha.
